docs/ref/models/fields.txt

```diff
@@ -1302,8 +1302,8 @@ materialized view.
     real column. If ``False``, the column acts as a virtual column and does
     not occupy database storage space.
 
-    PostgreSQL only supports persisted columns. Oracle only supports virtual
-    columns.
+    PostgreSQL supports persisted and virtual columns.
+    Oracle only supports virtual columns.
```

Comment:
The comment should mention that virtual columns are only supported on 18+ on Postgres.

Reply:
I updated the documentation.